docs/devices/X5H-GB-B.md

```diff
@@ -42,6 +42,11 @@ This climate device supports the following features: `current_heating_setpoint`,
 - `local_temperature_calibration`: Offset to be used in the local_temperature. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"local_temperature_calibration": VALUE}.`
 
 ### Schedule (text)
+There are 8 periods in the schedule in total. 6 for workdays and 2 for holidays. It should be set in the following format for each of the periods:
```

Comment:
This part is generated based on zigbee-herdsman-converters so it should not be edited manually. Please add to https://github.com/Koenkk/zigbee-herdsman-converters/blob/a731b4aacf0936b1194caef90fd453ec34802c47/devices/tuya.js#L2472 

```js
exposes.text('schedule', ea.STATE_SET).withDescription("DESCRIPTION HERE")
```